packages/css/src/__tests__/transform.test.tsx

```diff
@@ -25,7 +25,7 @@ describe('#css-transform', () => {
       const { sheets: actual } = transformCss(`
         margin-left: initial;
         content: 'hello';
-        color: hotpink;
```

Comment:
What's wrong with `hotpink`? 😅

Reply:
Original comment is here https://github.com/atlassian-labs/compiled/pull/837#discussion_r717566094